simapp/app.go

```diff
@@ -230,7 +230,7 @@ func NewSimApp(
 	memKeys := sdk.NewMemoryStoreKeys(capabilitytypes.MemStoreKey, "testingkey")
 
 	// load state streaming if enabled
-	if _, _, err := streaming.LoadStreamingServices(bApp, appOpts, appCodec, keys); err != nil {
+	if _, _, err := streaming.LoadStreamingServices(bApp, appOpts, appCodec, logger, keys); err != nil {
```

Comment:
any chance we can avoid this api break in the backport to 0.46 and 0.45?

Reply:
It's only to output an error log when `stopNodeAtError=false`, in my use case, I'll probably always set it to `true`, so I'm ok to remove it.